scans/auth/auth_plan_tests.sh

```diff
@@ -50,6 +56,19 @@ runplan()
     fi
 }
 
+getreportdetails()
+{
+    AUTHREPORT = $1
```

Comment:
I'm getting a failure `/zap/wrk/scans/auth/auth_plan_tests.sh: line 61: AUTHREPORT: command not found`
I think its the spaces around the `=` - they can cause problems

Reply:
Fix coming.